lib/jsdom/living/nodes/HTMLOutputElement-impl.js

```diff
@@ -23,6 +24,14 @@ class HTMLOutputElementImpl extends HTMLElementImpl {
   get form() {
     return formOwner(this);
   }
+
+  get defaultValue() {
+    return childTextContent(this);
+  }
+
+  set defaultValue(val) {
+    this.textContent = val;
+  }
```

Comment:
This code is missing some steps from the [spec](https://html.spec.whatwg.org/multipage/form-elements.html#dom-output-defaultvalue), and I don't think it makes sense to implement `defaultValue` without having `value` at the same time. It'd be great if you'd like to do that too, but otherwise you could add a dont-upstream version ([example](https://github.com/jsdom/jsdom/blob/master/test/web-platform-tests/to-upstream/dom/nodes/Element-closest-dont-upstream.html)) of the `reset-form.html` test with the `<output>`-specific tests disabled for now.

Reply:
Added `value` attribute to output element, and fixed up the implementation to satisfy `value`, `defaultValue`, and form reset steps mentioned in that spec. I also added the `type` type attribute for posterity, and enabled some of the output element web-platform-tests in `html/semantics/forms/the-output-element`. Hopefully this is helpful =)